.github/copilot-instructions.md

```
@@ -1,32 +1,51 @@
-- シンプルな関数型プログラミングを心掛ける
+### 技術スタック
+
+- Next.js 15
+- Chakra UI v3
+- Nivo charts
 
-## Critical Patterns
+### サイト構成
 
-### Error Handling Pattern
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)